docs/maintainer-workflow.md

```diff
@@ -0,0 +1,70 @@
+# Maintainer guide
+
+Welcome to the maintainer guide for our project!
+As a project maintainer, you play a crucial role in ensuring the success and health of the project.
+Below are key responsibilities and guidelines to help you manage the project effectively.
+
+## Key responsibilities
+
+### Manage Issues
+1. Issue Triage
+    - Establish a regular schedule for reviewing and triaging open issues.
```

Comment:
A good idea could be triaging issues using some chunk of time off of the community meetings.

Reply:
Adding this in community meeting document.